src/utils/api.js

```diff
@@ -39,4 +39,18 @@ export const postComment = (props, givenUsername, givenBody) => {
     .catch((err) => {
         return err
     })
+}
+
+export const fetchAllTopics = () => {
+    return myApi.get(`/topics`)
+    .then(({data}) => {
+        return data
+    })
+}
+
+export const fetchArticlesByTopic = (props) => {
```

Comment:
You should reuse the fetchAllArticles function using 'axios params'.  See NC notes/Axios/Axios Params.